.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: /
+  schedule:
+    interval: daily
```

Comment:
Can I ask if there's a reason for this frequency? It can potentially be very noisy, no?

Reply:
On second thoughts, I think it should be fine.

Reply:
My idea here is to start with more frequent so that we get more experience with this sooner.  I would not be surprised if we later settle on weekly as the best schedule, but seeing as this is a little experimental, it's not ideal if it only does something once a week right now.